docs/src/main/sphinx/sql/select.md

```diff
@@ -69,6 +69,31 @@ ROLLUP ( column [, ...] )
 
 Retrieve rows from zero or more tables.
 
+(select-with-session)=
```

Comment:
Can we add a reference here from https://trino.io/docs/current/sql/set-session.html ?

Reply:
Sure .. sent a PR .. probably worth a sentence in there and a link in See also at the bottom.